docs/docs/09.2-package-management.md

```diff
@@ -0,0 +1,123 @@
+---
+id: package-management
+title: Package Management
+permalink: package-management.html
+prev: language-tooling.html
+next: addons.html
+---
+
+## CDN-hosted React
+
+We provide CDN-hosted versions of React [on our download page](/react/downloads.html). These pre-built files use the UMD module format. Dropping them in with a simple `<script>` tag will inject the `React` and `ReactDOM` globals into your environment. It should also work out-of-the-box in CommonJS and AMD environments.
+
+
+## Using React from npm
+
+You can use React with a CommonJS module system like [browserify](http://browserify.org/) or [webpack](https://webpack.github.io/). Use the [`react`](https://www.npmjs.com/package/react) and [`react-dom`](https://www.npmjs.com/package/react-dom) npm packages.
+
+```js
+// main.js
+var React = require('react');
+var ReactDOM = require('react-dom');
+
+ReactDOM.render(
+  <h1>Hello, world!</h1>,
+  document.getElementById('example')
+);
+```
+
+Configure [babel](https://babeljs.io/) with a `.babelrc` file:
+ 
+ ```json
+ { "presets": ["react"] }
+ ```
+ 
+ > Note:
+ >
+ > If you are using ES2015, you will want to also use the `babel-preset-es2015` package.
+
+
+To install React DOM and build your bundle with browserify:
+
+```sh
+$ npm install --save react react-dom babelify babel-preset-react
+$ browserify -t [ babelify ] main.js -o bundle.js
+```
+
+To install React DOM and build your bundle with webpack:
+
+```sh
+$ npm install --save react react-dom babel-preset-react babel-loader babel-core
+$ webpack main.js bundle.js --module-bind 'js=babel-loader?presets=react'
```

Comment:
Since we are using `.babelrc` looks like we need to change this to just use `--module-bind 'js=babel-loader'`.